app/controllers/dinkurs_events_controller.rb

```diff
@@ -0,0 +1,116 @@
+class DinkursEventsController < ApplicationController
```

Comment:
Similar to above comment, you might want to name this `company_events_controller`, since it is preferable to have one controller for Dinkurs and local (non-Dinkurs) events.

I would also think about whether events should be nested resources within `Company`.  I think that nesting allows a conceptual rationale for dealing with both types of events within the same controller.